README.md

```diff
@@ -1,25 +1,78 @@
 # HA Cluster bootstrap salt formula
 
-Formulas for bootstrapping and managing a high availability cluster
-using crmsh.
+Salt formula to bootstrap and manage a [ClusterLabs](https://clusterlabs.org/) high availability cluster.
 
-Mainly adapted to SUSE / openSUSE Linux distributions, but should be
-usable on other distributions with minor modifications.
+Mainly adapted to SUSE / openSUSE Linux distributions (it is based in
+[crmsh](https://github.com/ClusterLabs/crmsh)), but it should be usable on other distributions with
+some modifications.
+
+## Features
+
+The formula provides the capability to create and configure a multi node HA cluster. Here some
+features:
+- Initialize a cluster
+- Join a node to an existing cluster
+- Remove a node from an existing cluster
+- Configure the pre-requirements (install required packages, configure `ntp/chrony`, create ssh-keys, etc)
+- Auto detect if the cluster is running in a cloud provider (Azure, AWS or GCP)
+- Configure sbd
+- Configure corosync
+- Configure the resource agents
+- Install and configure the [ha_cluster_exporter](https://github.com/ClusterLabs/ha_cluster_exporter)
+
+## Installation
+
+The project can be installed in many ways, including but not limited to:
+
+1. [RPM](#rpm)
+2. [Manual clone](#manual-clone)
+
+### RPM
+
+On openSUSE or SUSE Linux Enterprise you can just use the `zypper` system package manager:
+```shell
+zypper install habootstrap-formula
+```
+
+**Important!** This will install the formula in `/usr/share/salt-formulas/states/cluster`. Make sure that `/usr/share/salt-formulas/states` entry is correctly configured in your salt minion configuration `file_roots` entry if the formula is used in a masterless mode.
```

Comment:
```suggestion
**Important!** This will install the formula in `/usr/share/salt-formulas/states/cluster`. Make sure that `/usr/share/salt-formulas/states` entry is correctly configured in your Salt minion configuration `file_roots` entry if the formula is used in a masterless mode.
```